### ISSUE TITLE
Create multiple sitemaps when there are many url

### ISSUE THREAD

**Issue by esedArk** [spatie/laravel-sitemap#311]
A good practice is to create a sitemap every 1000 urls, when a page has 30 or 40 thousand urls, it would be best to create 1 sitemap index and as many child sitemap as 1000 urls have the web page.

**Comment by sarzali07** [comment]
> A good practice is to create a sitemap every 1000 urls, when a page has 30 or 40 thousand urls, it would be best to create 1 sitemap index and as many child sitemap as 1000 urls have the web page.

Did you find any solution??

**Comment by ametad** [comment]
Read the docs, please...

TL;DR laravel-sitemap by default create multiple sitemap files for every 50000 urls. It is configurable.

**Comment by szabizs** [comment]
> Read the docs, please...
> 
> TL;DR laravel-sitemap by default create multiple sitemap files for every 50000 urls. It is configurable.

Where do exactly we find this info in the documentation?

**Comment by ametad** [comment]
With documentation I also mean the source code: https://github.com/spatie/laravel-sitemap/blob/6676631f2768f3476fb3e1f37ea9cc203dd1317e/src/SitemapGenerator.php#L84

**Comment by ctf0** [comment]
for future reference, the package check for `$maximumTagsPerSitemap` which by default is set to `false` and so to leverage it, u just need to add `->maxTagsPerSitemap()` & the package will take care of splitting the file into indexes
